### PR TITLE
feat: badge component

### PR DESCRIPTION
**Pull request by Rohitprasad83**
badge

### CODE REVIEW

**Review by maitrakhatri — APPROVED** [review]
looks perfect to me

File: badge/badge.html [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+    <meta name="description" content="Badge" />
+    <title>Badge</title>
+    <link rel="stylesheet" href="./badge.css" />
+  </head>
+  <body>
+    <h1>Avatar with badge</h1>
+    <div class="container">
+      <div class="avatar__with__badge">
+        <div class="avatar">
+          <img src="https://img.icons8.com/ios-glyphs/90/000000/user--v1.png" />
```
Comment (Aditi0505) [inline badge/badge.html]: Suggestion: Can add alt attribute for accessibility.

**Review by Aditi0505 — APPROVED** [review]
Have added a suggestion to use alt attribute. This is missing in multiple places. Requesting you to fix it so that it passes the accessibility check.

File: badge/badge.html (hunk 1 above)
Comment (Rohitprasad83) [reply badge/badge.html]: Thanks for pointing it out.

**Pull request merged**